pyfakefs/fake_pathlib.py

```diff
@@ -53,8 +53,9 @@ def init_module(filesystem):
 
 def _wrap_strfunc(strfunc):
     @functools.wraps(strfunc)
-    def _wrapped(pathobj, *args):
-        return strfunc(pathobj.filesystem, str(pathobj), *args)
+    def _wrapped(pathobj, *args, **kwargs):
+        p = pathobj.filesystem if hasattr(pathobj, "filesystem") else pathobj
```

Comment:
This change was actually not needed - it happened because the fake accessor was not set (why I had to implement the two class methods to call `_init`).